sdk/javascript/src/express_endpoint.ts

```diff
@@ -1,24 +1,56 @@
 import express from 'express';
 import {encode, sign} from './reporter';
 
+type UnixT = number;
+type Pairs = object | [any, any][];
+type Query = any;
+
+function flatStringList(param: string | string[]): string[] {
+  if (Array.isArray(param))
+    return param.reduce((a: string[], s) => a.concat(flatStringList(s)), []);
+  return param.split(',');
+}
+
+function upper(str: string): string {
+  return str.toUpperCase();
+}
+
+export const Filters = {
+  symbols: (pairs: Pairs, query: Query): Pairs => {
+    if (query.symbols) {
+      const symbols = new Set(flatStringList(query.symbols || []).map(upper));
+      if (Array.isArray(pairs))
+        return pairs.filter(([k, _]) => symbols.has(upper(k)));
+      return Object.entries(pairs).reduce((a, [k, v]) => {
```

Comment:
kind of lost on what the pair param is

Reply:
like it's an object or an array?, also it is compared to the symbols filter?

Reply:
Pairs is either a list of key, values or an object (was/is allowed by encoder) - this is to preserve whatever the input form given was on the filtered output - on the one hand kind of sucks to have this code but otoh much nicer to match what was given IMO